### PR TITLE
Review payout schemas (CS1/CS2)

### PR DESCRIPTION
**Pull request by martinseco**

### CODE REVIEW

File: payments/abc/destinations.go [hunk 1]
```diff
@@ -8,45 +8,57 @@ import (
...
-		Type           payments.PaymentDestinationType `json:"type,omitempty"`
-		Number         string                          `json:"number,omitempty"`
-		ExpiryMonth    int                             `json:"expiry_month,omitempty"`
-		ExpiryYear     int                             `json:"expiry_year,omitempty"`
-		FirstName      string                          `json:"first_name,omitempty"`
-		LastName       string                          `json:"last_name,omitempty"`
-		Name           string                          `json:"name,omitempty"`
-		BillingAddress *common.Address                 `json:"billing_address,omitempty"`
-		Phone          *common.Phone                   `json:"phone,omitempty"`
+	requestCardDestination struct {
```
Comment (armandojaleo) [inline payments/abc/destinations.go:11]: Why in lowercase?

Comment (armandojaleo) [reply payments/abc/destinations.go:11]: 👍

**Comment by armandojaleo** [comment]
LGTM!

**Review by Armando-Ibarra — APPROVED** [review]
Very good job!

**Pull request merged**